app/src/main/res/menu/menu_bottom_view_pager.xml

```diff
@@ -30,7 +30,7 @@
         android:title="@string/details"/>
 
     <item
-        android:id="@+id/action_favourites"
+        android:id="@+id/action_favourites2"
```

Comment:
Why is this called `action_favourite2` and not `action_favourite`?

Reply:
As we have two favorite menu items, one for the LFMainActivity and one for the SingleMediaActivity while testing had changed the name to avoid confusion, but forgot to change it back to action_favourites. Will revert it to action_favourites now.